Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -48,13 +48,25 @@ namespace gdjs {
 
         // Get associated behaviors
         const behaviorA = contact.GetFixtureA().GetBody()
-          .gdjsAssociatedBehavior;
+          .gdjsAssociatedBehavior as Physics2RuntimeBehavior;
         const behaviorB = contact.GetFixtureB().GetBody()
-          .gdjsAssociatedBehavior;
+          .gdjsAssociatedBehavior as Physics2RuntimeBehavior;
 
-        // Let each behavior know about the contact against the other
-        behaviorA.currentContacts.push(behaviorB);
-        behaviorB.currentContacts.push(behaviorA);
+        // There might be contacts that end during the frame and
+        // start again right away. It is considered a glitch
+        // and should not be detected.
+        let i = behaviorA.contactsEndedBetweenFrames.indexOf(behaviorB);
+        if (i !== -1) {
+          behaviorA.contactsEndedBetweenFrames.splice(i, 1);
+        } else {
+          behaviorA.contactsStartedBetweenFrames.push(behaviorB);
+        }
```

Comment:
It could be a `onContactBegin` method in the behavior.

Reply:
Great idea, I'll do this